website/demos/ContextMenu.tsx

```diff
@@ -129,6 +129,7 @@ function rowKeyGetter(row: Row) {
 
 function RowRenderer(props: RowRendererProps<Row>) {
   return (
+    // @ts-expect-error
     <ContextMenuTrigger id="grid-context-menu" collect={() => ({ rowIdx: props.rowIdx })}>
```

Comment:
May be we can remove ContextMenu library and add a basic popup for examples.

Reply:
I will check

Reply:
Preferably yes, the library isn't maintained anymore.